cmd/cmds_test.go

```diff
@@ -0,0 +1,366 @@
+package fscmds_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	fscmds "github.com/djdv/go-filesystem-utils/cmd"
+	cmds "github.com/ipfs/go-ipfs-cmds"
+	"github.com/multiformats/go-multiaddr"
+)
+
+var root = &cmds.Command{
+	Options: fscmds.Options,
+	Helptext: cmds.HelpText{
+		Tagline: "File system service ⚠[TEST]⚠ utility.",
+	},
+}
+
+func TestRequestsStandard(t *testing.T) {
```

Comment:
This function is way too big. Needs to be split up.

Reply:
New: https://github.com/djdv/go-filesystem-utils/pull/2/commits/09177f7f0bd99f1708b7224b712432157f26bd51